jobs/awslogs/templates/bin/pre-start

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+echo "[general]"
+echo "state_file = /var/vcap/sys/log/awslogs/agent-state"
+
+for SYSTEM_LOG in /var/log/audit/audit.log /var/log/auth.log /var/log/dpkg.log /var/log/messages /var/log/syslog
```

Comment:
Because I thought this script would be replacing the existing file, so I took pains to reconstitute the log file as it stood and then add the rest.